### ISSUE TITLE
Artist and title get lost when setting max height

### ISSUE THREAD

**Issue by davidrustingha** [custom-cards/spotify-card#119]
**Describe the bug**
When setting a max height for the Spotify card the currently playing artist and title disappear. (picture 1) 
When I click 'edit dashboard' they are visible, but only a little bit (picture 2)
Why would I want a max height: I have a lot of playlists and want to show them in a relatively small box (500px as max height). That way I can just scroll down to see the rest of the playlists.

* [X] Reproducable on a plain lovelace panel without card, style or theme mods
* [X] Using latest version of spotcast and spotify-card [ ]
* [X] Using latest stable version of Home Assistant
* [X] I have Spotify Premium
* [X] I have setup the Spotify integration in Home Assistant

**To Reproduce**
1. Go to dashboard
2. Configure spotify card
3. Set max height

**Expected behavior**
I expected to see the currently playing text.

**Screenshots**
![Screenshot_1](https://user-images.githubusercontent.com/62120423/97116613-a45d1500-16fe-11eb-8323-829499143771.jpg)
![Screenshot_2](https://user-images.githubusercontent.com/62120423/97116614-a4f5ab80-16fe-11eb-83f3-8ca702ffdac9.jpg)

**Environment (please complete the following information):**
 - OS: Windows, iOS, Android
 - Browser: Chrome, Opera Mobile
 - Version: Latest
 - HA version: Latest stable
 - spotcast version: Latest stable

**Configuration**
```
type: 'custom:spotify-card'
account: david
always_play_random_song: true
height: 500
limit: 50
hide_top_header: true
```

**Comment by FL550** [comment]
Thanks for the report! I know the reason for this issue, my bad. I will fix this tomorrow

**Comment by FL550** [comment]
This should now be fixed in [v2.3.4](https://github.com/custom-cards/spotify-card/tree/v2.3.4).

**Comment by davidrustingha** [comment]
Yes, awesome!
